cmd/syncthing/cli/await_insync.go

```diff
@@ -0,0 +1,98 @@
+// Copyright (C) 2021 The Syncthing Authors.
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+package cli
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"time"
+
+	"github.com/urfave/cli"
+)
+
+var awaitInsyncCommand = cli.Command{
+	Name:     "await-insync",
+	HideHelp: true,
+	Usage:    "Block until the current dir finishes syncing",
```

Comment:
I think "in current dir" is kind of special-case. I would expect await-insync to take a list of one or more folder IDs and wait for those folders, and perhaps have a shortcut `--current-dir` to do what it does currently. (I don't think it would be much more code either.)